src/main/java/org/opensrp/web/dto/TaskSearchCriteria.java

```diff
@@ -0,0 +1,108 @@
+package org.opensrp.web.dto;
+
+import javax.validation.constraints.NotBlank;
+import java.util.List;
+
+public class TaskSearchCriteria {
```

Comment:
Can we just use TaskSearchBean instead of adding a new object

Reply:
Sure. Added validation in TaskSearchBean, here is a core PR for it : https://github.com/OpenSRP/opensrp-server-core/pull/380